userdatamodel/user.py

```diff
@@ -82,27 +82,18 @@ class User(Base):
 
     google_proxy_group = relationship(
         "GoogleProxyGroup",
-        backref=backref(__tablename__, uselist=False, cascade="all, delete-orphan"),
+        backref=backref(
+            __tablename__,
+            uselist=False,
+            cascade="save-update, merge, refresh-expire, expunge",
```

Comment:
why is this one different?

Reply:
Because User is actually a child of GoogleProxyGroup (that is to say User has a fkey column pointing to GoogleProxyGroup.id), and it doesn't necessarily make sense to automatically delete the User upon deleting their GPG